scripts/sysman/rasState.yml

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Ras Get State"
+version: "1.7"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "RAS Get State Extension Name"
+version: "1.7"
+name: $S_RAS_GET_STATE_EXT_NAME
+value: '"$XS_extension_ras_state"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "RAS Get State Extension Version(s)"
+version: "1.7"
+name: $s_ras_state_ext_version_t
+etors:
+    - name: "1_7"
```

Comment:
this is the version of the extension, which is 1.0.

Reply:
fixed